src/main/java/com/reimbursements/requests/CreateUserRequest.java

```diff
@@ -1,21 +1,14 @@
-package com.revature.requests;
+package com.reimbursements.requests;
+
 public class CreateUserRequest {
 	
     private String username;
     private String password;
     private String firstname;
     private String lastname;
     private String email;
-    private int roleID;
+    private String role;
     private int authorityID;
-   

```

Comment:
Missing a constructor with two parameters. It's a good way of creating an object, to pass those parameters. Eventually `Lombok`s `@Data` annotation is a good option, if you understand and accept what Lombok does underneath, and what the risk.